app/views/dashboards/_table_content.html.erb

```diff
@@ -85,7 +85,7 @@
     </tbody>
   </table>
 
-  <% if table_type == 'call_list' && current_user.patients.count.nonzero? %>
+  <% if table_type == 'call_list' && current_user.call_lists.count.nonzero? %>
```

Comment:
A surprising amount of this pr was some variation of `user.patients` => `user.call_lists`.